### DISCUSSION TITLE
Słownik

### DISCUSSION THREAD

**Discussion by olsza** [davpal/progit2-pl#23] [Ideas]
Zastanawiam się o stworzeniu słownika, co też wspomniał @davpal 

Ja myślałem np. utworzyć Wiki i tam wskazówki, ale myślę by sięć tropem jak to ma [DE](https://github.com/progit/progit2-de/blob/main/TRANSLATION_NOTES_DE.asc) czyli utworzeniu słownika w pliku.

Jak myślicie @davpal @lkord , która opcja lepsza? lub może jeszcze coś nowego?

**Comment by lkord** [comment]
@olsza mysle że plik. Wiki chyba się nie ściąga z repo?

**Reply by olsza** [reply]
Prawda

**Comment by olsza** [comment]
Stosowane terminologii polskiej to prawdopodobnie będziemy używać jak jest w GIT, czyli https://github.com/git/git/blob/master/po/pl.po
Aby było wszystko spójne... ale to będzie jeszcze kwestia do obgadania.

**Reply by olsza** [reply]
Zastanawiam się czy w książce nie powinniśmy używać i polskiego i w nawiasach angielskich nazw.
Git-a w 90% używają osoby z IT i raczej mówi się np. "zrób commit-a" a nie "zrób zapis",ale aby książka była nie tylko przyjazna dla osób IT,ale dla innych i mysle byśmy np. tłumaczyli np. "trzeba zrobić zapis (commit)" 

Co o tym myślicie?

Pytam,bo zanim będą robione poprawki w  tłumaczeniach, aby ustalić jakiś reguły,bo przy aktualnym stanie co przeglądam,to wszytko zależy kto tłumaczył to było inaczej, a chce aby było ujednolicone.

**Reply by lkord** [reply]
Jestem za, też tak czułem jak tłumaczylem, że więcej zamieszania to wprowadza. Po angielsku jest jednoznacznie.

**Comment by olsza** [comment]
Czy ktoś przygotuje taki plik jak podałem wyżej przekład w DE i przygotuje dla PL ?
Dobrze by było uwzględnić jak jest tłumaczenie w opisach komend GIT (link też wyżej podałem) dane słowa/wyrażenie a jakie obecnie w książce lub dać swoje propozycje,ktore zaglosujemy i zdecydujemy 😉

**Reply by lkord** [reply]
W wersji niemieckiej progit2 jest taki plik
https://github.com/progit/progit2-de/blob/main/TRANSLATION_NOTES_DE.asc
To który bierzemy za wzór? Sama tabelka z tego linka co tu daję wystarczy? @olsza jak uważasz?
